litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/index.tsx

```diff
@@ -222,8 +231,18 @@ const TuneWorkflow = forwardRef((_, ref) => {
       .then((data) => {
         data.text().then((yamlText) => {
           const wfmanifest = updateEngineName(YAML.parse(yamlText));
+          const updatedManifest = YAML.parse(wfmanifest);
+          updatedManifest.spec.arguments.parameters.forEach(
+            (parameter: any, index: number) => {
+              if (parameter.name === 'adminModeNamespace') {
```

Comment:
Can you check if it's possible/required to convert 'adminModeNamespace' into a constant? 🤔